.changeset/pretty-pants-end.md

```diff
@@ -0,0 +1,6 @@
+---
+"kilo-code": minor
+"@roo-code/types": patch
```

Comment:
```suggestion
```

We [don't update](https://kilo-code.slack.com/archives/C08HC5A2EDT/p1764890523963609) the version of roo inherited packages.